scripts/download-signature-key.sh

```diff
@@ -46,7 +41,13 @@ VERSION="$2"
 # package will not be correct. When this package is built by our build
 # system and automation, these variables should be available.
 #
-if [[ -n "${DELPHIX_SIGNATURE_TOKEN:-}" ]] && [[ -n "${DELPHIX_SIGNATURE_URL:-}" ]]; then
+if [[ -n "${DELPHIX_SIGNATURE_TOKEN:-}" ]] &&
+	[[ -n "${DELPHIX_SIGNATURE_URL:-}" ]] &&
+	[[ -n "${DELPHIX_SIGNATURE_VERSION:-}" ]]; then
```

Comment:
It's worth point out, that we pass in `DELPHIX_SIGNATURE_VERSION` as a parameter to this script.

If we're going to check that this variable is set here, it might make sense to more simply remove the second parameter to this script, and instead use `DELPHIX_SIGNATURE_VERSION` instead of `VERSION`.

I don't think it's a big deal either way.. it won't practically change this script.. but it just struck me as odd, that we check for `DELPHIX_SIGNATURE_VERSION` here, but we also pass that variable in by the caller, and then use `VERSION` later, which happens to be derived from `DELPHIX_SIGNATURE_VERSION`.

Reply:
Yeah thought about that, but felt like there are 2 workflows eventually invoking this - manual build (like a developer running make packages etc) where `VERSION` sounds relevant and the automated jenkins jobs where `DELPHIX_SIGNATURE_VERSION` becomes relevant.
So prefer to leave this as is for now.